stratisd_cert.py

```diff
@@ -823,6 +823,23 @@ def test_pool_set_fs_limit_too_low(self):
             "org.freedesktop.DBus.Error.Failed",
         )
 
+    @skip(_skip_condition(1))
+    def test_blockdev_set_userinfo_property(self):
+        """
+        Test setting the UserInfo property on a block device.
+        """
+        pool_name = p_n()
+        blockdev_path = StratisCertify.DISKS[0:1]
+        _, blkdev_paths = make_test_pool(pool_name, blockdev_path)
+
+        self._unittest_set_property(
+            blkdev_paths[0],
+            StratisDbus.BLKDEV_IFACE,
+            "UserInfo",
+            (True, dbus.String("eeeeeeeeee")),
```

Comment:
For consistency, you should wrap the ```True``` in a ```dbus.Boolean``` constructor call, as we do w/ all other boolean values we put on the D-Bus.

Reply:
Resolved.